Client/results.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="results.css" />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Google Results</title>
+  </head>
+  <body>
+    <h1>Results</h1>
```

Comment:
This is a nice temporary fix! You guys could add a button to take you back to the google homepage. It would be easier for the user to navigate around the website :)